### ISSUE TITLE
Install step 'setup.sh start' fails with "Unable to locate package python-flask"

### ISSUE THREAD

**Issue by greiginsydney** [greiginsydney/Intervalometerator#102]
A new install or upgrade fails shortly after commencing the `sudo -E -H ./setup.sh start` step:

```bash
Installing python3-pip, python-flask
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
E: Unable to locate package python-flask
greig@BenchBlackPCB:~ $
```

![image](https://user-images.githubusercontent.com/11004787/141074302-e923a8c9-5cf2-4449-8c37-3d7529080809.png)

**Comment by greiginsydney** [comment]
It seems that the flask installer has changed. It's now `python3-flask`.

If you edit the setup script (`sudo nano setup.sh`), change line 120 to now say:
```bash
apt-get install python3-pip python3-flask -y
```
![image](https://user-images.githubusercontent.com/11004787/141074739-0a1c413a-d30c-4a6c-b6d6-00bc40896477.png)

... and you'll be right. I'll fix this in the next release.

\- Greig.

(Thanks Dave for bringing this to my attention).

**Comment by greiginsydney** [comment]
Fixed in [release 4.4.0](https://github.com/greiginsydney/Intervalometerator/releases/tag/v4.4.0)
